ksqldb-udf-quickstart/src/main/resources/archetype-resources/pom.xml

```diff
@@ -61,7 +61,7 @@
                 <configuration>
                     <compilerArgs>
                         <arg>-Xlint:all</arg>
-                        <arg>-Werror</arg>
+                        <arg>${compile.warnings-flag}</arg>
```

Comment:
i think this one should stay as it's an archetype resource (so it isn't built through the usual ksql build); we could also remove it entirely but let's punt that for another time